ZipBuilder/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -50,7 +50,20 @@ public enum CocoaPodUtils {
     }
   }
 
-  /// Execute the `pod cache list` command to get the Pods currently cached on your machine.
```

Comment:
This is a bit of a poor diff - the below function had a duplicate at the start, I suspect a copy/paste error. Unrelated to this change, but I removed the duplicate.